opyn/opyn/settlement.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------
+# Created By: Haythem@Opyn, Anil@Opyn
+# Created Date: 04/04/2022
+# version ='0.1.0'
+# ---------------------------------------------------------------------------
+""" Module to call Swap contract """
+# ---------------------------------------------------------------------------
+
+# ---------------------------------------------------------------------------
+# Imports
+# ---------------------------------------------------------------------------
+from web3 import Web3
+from dataclasses import asdict
+from opyn.contract import ContractConnection
+from opyn.utils import get_address
+from opyn.definitions import OrderData
+
+# ---------------------------------------------------------------------------
+# Settlement Contract
+# ---------------------------------------------------------------------------
+class SettlementContract(ContractConnection):
+    """
+    Object to create connection to the Settlement contract
+
+    Args:
+        config (ContractConfig): Configuration to setup the Contract
+    """
+
+    def settleRfq(self, publicKey: str, privateKey: str, _offerOrder: OrderData, _bidOrder: OrderData):
```

Comment:
@haythem96 maybe we should remove this from here as this is settle is directly called by partner and not something used by friends at paradigmco . do you agree with that @pdonorio ?

Reply:
Yes, anything that is not needed by paradigm's flow may be in your own repo or where you'd prefer 👍🏽